changelog.txt

```diff
@@ -1,10 +1,22 @@
-=======
+======
 Changes
 =======
 
 The **signac-flow** package follows `semantic versioning <https://semver.org/>`_.
 The numbers in brackets denote the related GitHub issue and/or pull request.
 
+next
+====
+
+next
+----
+
+Removed
++++++++
+
+- Dropped support for Python 3.5 (#305). The signac project will follow the `NEP 29 deprecation policy <https://numpy.org/neps/nep-0029-deprecation_policy.html>`_ going forward.
```

Comment:
```suggestion
- Drop support for Python 3.5 (#305). The signac project will follow the `NEP 29 deprecation policy <https://numpy.org/neps/nep-0029-deprecation_policy.html>`_ going forward.
```
Changelog entries should be in present tense, see other entries (mostly consistent).